README.md

```diff
@@ -42,6 +41,36 @@ api = Api(app, '/api')
 api.add_resource('/widgets', WidgetListView, WidgetView)
 ```
 
+By default, models are expected to have been created using [Flask-SQLAlchemy](http://flask-sqlalchemy.pocoo.org/).
+
+```python
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import Column, String
+
+from . import app
+
+db = SQLAlchemy(app)
+
+
+class Widget(db.Model):
+    id = Column(String, primary_key=True)
+    name = Column(String, nullable=False)
+    color = Column(String, nullable=False)
+```
+
+Schemas can be standard marshmallow `Schema` instances or [marshmallow-sqlalchemy](https://marshmallow-sqlalchemy.readthedocs.io/) `TableSchema` instances. They should not be `ModelSchema` instances.
```

Comment:
feel free to offer this if you think it is helpful. I do like it in that I can point to a model rather than a table:
```
class LiteModelSchema(with_metaclass(ModelSchemaMeta, ma.Schema)):
    OPTIONS_CLASS = ModelSchemaOpts
```

Reply:
let's leave that for later... like for the more comprehensive docs